### DISCUSSION TITLE
Document formatting with AI

### DISCUSSION THREAD

**Discussion by Kotik001** [XInTheDark/raycast-g4f#110] [Ideas]
Hello! Like your project! My idea is to let models have some documents (pdf, word) as input, ask them to change some formatting or content and give files output. Is it achievable? Thank you so much in advance!

**Comment by XInTheDark** [comment]
Hi - I'm currently looking into file output (e.g. Artifacts feature in some websites)! by the way, there is a Format Text command available, and you can also build your own Custom AI Command in the mean time :)
